crowbar_framework/spec/controllers/crowbar_controller_spec.rb

```diff
@@ -234,8 +234,8 @@
     # missing nodes. However, this is ok, as users will assign roles to them
     # later.
     before(:each) do
-      CrowbarService.any_instance.expects(:validate_proposal).at_least_once
-      CrowbarService.any_instance.expects(:validate_proposal_elements).returns(true).at_least_once
+      expect_any_instance_of(CrowbarService).to receive(:validate_proposal)
+      expect_any_instance_of(CrowbarService).to receive(:validate_proposal_elements).and_return(true)
```

Comment:
Metrics/LineLength: Line is too long. [101/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)